contributing.md

```diff
@@ -62,6 +62,9 @@ If you have an incompatible version of Node in your development environment, you
 nvm install
 ```
 
+A [Google Fonts API Key](https://developers.google.com/fonts/docs/developer_api)  is required to build or update Google font list. May not be require for many contributors.  
```

Comment:
```suggestion
A [Google Fonts API Key](https://developers.google.com/fonts/docs/developer_api)  is required to build or update Google font list. May not be required for many contributors.  
```

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3110/commits/1c057e221ad70b2f4c3cf421f7faee74a000331a